README.md

```diff
@@ -6,35 +6,38 @@ Just like everyone can use mind mapping software to organize ideas and improve e
 
 XMindMark files are written in plain text, which means that you can describe a mind map faster using any text editor at any time anywhere, like:
 
-```
+```XMindMark
```

Comment:
Thanks for your suggestion, but I think lower-case version `xmindmark` will be better, for all similar code blocks.